.vuepress/code-samples/generated-samples.json

```diff
@@ -76,9 +79,15 @@
       "search_guide_2": "$ curl --get 'http://localhost:7700/indexes/movies/search' \\\n  --data-urlencode 'q=Avengers' \\\n  --data-urlencode 'filters=release_date > 795484800'",
       "getting_started_create_index_md": "```bash\n  $ curl \\\n    -X POST 'http://127.0.0.1:7700/indexes' \\\n    --data '{\n    \"uid\" : \"movies\"\n  }'\n```",
       "getting_started_add_documents_md": "```bash\n  $ curl \\\n    -X POST 'http://127.0.0.1:7700/indexes/movies/documents' \\\n    --data @movies.json\n```",
-      "getting_started_search_md": "```bash\n$ curl \\\n  -X GET 'http://127.0.0.1:7700/indexes/movies/search?q=botman'\n```"
+      "getting_started_search_md": "```bash\n$ curl \\\n  -X GET 'http://127.0.0.1:7700/indexes/movies/search?q=botman'\n```",
+      "faceted_search_update_settings_1": "$ curl \\\n  -X POST 'http://localhost:7700/indexes/movies/settings' \\\n  --data '{\n      \"attributesForFaceting\": [\n          \"director\",\n          \"genres\"\n      ]\n  }'",
+      "faceted_search_facet_filters_1": "$ curl --get 'http://localhost:7700/indexes/movies/search' \\\n    --data-urlencode 'q=thriller' \\\n    --data-urlencode 'facetFilters=[[\"genres:Horror\", \"genres:Mystery\"], \"director:Jordan Peele\"]'",
+      "faceted_search_facets_distribution_1": "$ curl --get 'http://localhost:7700/indexes/movies/search' \\\n  --data-urlencode 'q=Batman' \\\n  --data-urlencode 'facetsDistribution=[\"genres\"]'",
+      "faceted_search_walkthrough_attributes_for_faceting_1": "$ curl \\\n  -X POST 'http://localhost:7700/indexes/movies/settings' \\\n  --data '{\n      \"attributesForFaceting\": [\n          \"director\",\n          \"producer\",\n          \"genres\",\n          \"production_companies\"\n      ]\n  }'",
+      "faceted_search_walkthrough_facet_filters_1": "$ curl --get 'http://localhost:7700/indexes/movies/search' \\\n    --data-urlencode 'q=thriller' \\\n    --data-urlencode 'facetFilters=[[\"genres:Horror\", \"genres:Mystery\"], \"director:Jordan Peele\"]'",
+      "faceted_search_walkthrough_facets_distribution_1": "$ curl --get 'http://localhost:7700/indexes/movies/search' \\\n    --data-urlencode 'q=Batman' \\\n    --data-urlencode 'facetsDistribution=[\"genres\"]'"
     },
     "language": "bash",
     "label": "cURL"
   }
-]
+]
```

Comment:
Ah I thought you made the code sample for the post route.

Could you change the samples in  `.code-samples.meilisearch.yaml` ? At the root of the directory